src/rules/__tests__/lowercase-name.test.ts

```diff
@@ -46,6 +46,7 @@ ruleTester.run('lowercase-name', rule, {
     'describe(``)',
     'describe("")',
     'describe(42)',
+    { code: 'describe(42)', options: [{ ignore: undefined }] },
```

Comment:
Does `defaultOptions` not work if the property is `undefined` instead of absent?

Reply:
if so, I think that should be reported as a bug

Reply:
So the problem here is coverage - You could argue this as a bug, but if we do, then we lose the ability to make coverage happy, as TS doesn't know about `defaultOptions`, and so will flag the properties as possibly undefined if we don't have the fallback value.

Reply:
I'm resolving this per my [comment here](https://github.com/jest-community/eslint-plugin-jest/pull/420#issuecomment-546645055).